github/resource_github_branch_protection_test.go

```
@@ -114,57 +134,67 @@ func TestAccGithubBranchProtection_teams(t *testing.T) {
-					resource.TestCheckResourceAttr("github_branch_protection.master", "required_pull_request_reviews.0.dismiss_stale_reviews", "true"),
-					resource.TestCheckResourceAttr("github_branch_protection.master", "required_pull_request_reviews.0.dismissal_users.#", "0"),
-					resource.TestCheckResourceAttr("github_branch_protection.master", "required_pull_request_reviews.0.dismissal_teams.#", "2"),
-					resource.TestCheckResourceAttr("github_branch_protection.master", "required_pull_request_reviews.0.require_code_owner_reviews", "false"),
-					resource.TestCheckResourceAttr("github_branch_protection.master", "restrictions.0.users.#", "0"),
-					resource.TestCheckResourceAttr("github_branch_protection.master", "restrictions.0.teams.#", "2"),
+					resource.TestCheckResourceAttr(rn, "repository", repoName),
+					resource.TestCheckResourceAttr(rn, "branch", "master"),
+					resource.TestCheckResourceAttr(rn, "enforce_admins", "true"),
+					resource.TestCheckResourceAttr(rn, "required_status_checks.0.strict", "false"),
+					resource.TestCheckResourceAttr(rn, "required_status_checks.0.contexts.#", "0"),
+					resource.TestCheckResourceAttr(rn, "required_pull_request_reviews.0.dismiss_stale_reviews", "true"),
+					resource.TestCheckResourceAttr(rn, "required_pull_request_reviews.0.dismissal_users.#", "0"),
+					resource.TestCheckResourceAttr(rn, "required_pull_request_reviews.0.dismissal_teams.#", "2"),
+					resource.TestCheckResourceAttr(rn, "required_pull_request_reviews.0.require_code_owner_reviews", "false"),
+					resource.TestCheckResourceAttr(rn, "restrictions.0.users.#", "0"),
+					resource.TestCheckResourceAttr(rn, "restrictions.0.teams.#", "2"),
 				),
 			},
+			{
+				ResourceName:      rn,
+				ImportState:       true,
+				ImportStateVerify: true,
+				// Possible API bug
+
+				ImportStateVerifyIgnore: []string{
+					"restrictions.0.teams.#",
```

We shouldn't ignore bugs, let the test fail so its apparently something needs to be fixed

---

Ok. Will change it back.